WordPressShared/WordPressShared.xcodeproj/project.pbxproj

```diff
@@ -659,6 +671,7 @@
 			baseConfigurationReference = 3BEDE0D11F9FF330B15F3EBB /* Pods-WordPressShared.release-internal.xcconfig */;
 			buildSettings = {
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = "$(inherited)";
+				CLANG_WARN_DOCUMENTATION_COMMENTS = NO;
```

Comment:
I don't think we want this (and all of its repetitions below), we have it on YES for the other projects.